<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesUnits</class>
 <widget class="QWidget" name="PreferencesUnits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_units">
     <property name="title">
      <string>Unit system</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>System</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="metric">
        <property name="text">
         <string>&amp;Metric</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="imperial">
        <property name="text">
         <string>Imperial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="personalize">
        <property name="text">
         <string>Personali&amp;ze</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="units_group">
     <property name="title">
      <string>Individual settings</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Depth</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="meter">
        <property name="text">
         <string>meter</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="feet">
        <property name="text">
         <string>feet</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pressure</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="bar">
        <property name="text">
         <string>bar</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="psi">
        <property name="text">
         <string>psi</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Volume</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="liter">
        <property name="text">
         <string>&amp;liter</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_3</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="cuft">
        <property name="text">
         <string>cuft</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_3</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="celsius">
        <property name="text">
         <string>celsius</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_4</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="fahrenheit">
        <property name="text">
         <string>fahrenheit</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_4</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Weight</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="kg">
        <property name="text">
         <string>kg</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_5</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QRadioButton" name="lbs">
        <property name="text">
         <string>lbs</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_5</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_time">
     <property name="title">
      <string>Time units</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Ascent/descent speed denominator</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="vertical_speed_minutes">
        <property name="text">
         <string>Minutes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="vertical_speed_seconds">
        <property name="text">
         <string>Seconds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_duration">
     <property name="title">
      <string>Duration units</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Show hours in duration</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="duration_show_hours">
        <property name="text">
         <string>hh:mm (always)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="duration_no_hours">
        <property name="text">
         <string>mm (always)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="duration_mixed">
        <property name="text">
         <string>mm (for dives shorter than 1 hour), hh:mm (otherwise)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_divelist">
     <property name="title">
      <string>Dive list table</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="show_units_table">
        <property name="text">
         <string>Show units in dive list table</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="show_mdecimal">
        <property name="text">
         <string>metric depth decimals</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_gps">
     <property name="title">
      <string>GPS coordinates</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Location Display</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="gpsTraditional">
        <property name="text">
         <string>traditional (dms)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="gpsDecimal">
        <property name="text">
         <string>decimal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>metric</tabstop>
  <tabstop>imperial</tabstop>
  <tabstop>personalize</tabstop>
  <tabstop>meter</tabstop>
  <tabstop>feet</tabstop>
  <tabstop>bar</tabstop>
  <tabstop>psi</tabstop>
  <tabstop>liter</tabstop>
  <tabstop>cuft</tabstop>
  <tabstop>celsius</tabstop>
  <tabstop>fahrenheit</tabstop>
  <tabstop>kg</tabstop>
  <tabstop>lbs</tabstop>
  <tabstop>vertical_speed_minutes</tabstop>
  <tabstop>vertical_speed_seconds</tabstop>
  <tabstop>duration_show_hours</tabstop>
  <tabstop>duration_no_hours</tabstop>
  <tabstop>duration_mixed</tabstop>
  <tabstop>gpsTraditional</tabstop>
  <tabstop>gpsDecimal</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>metric</sender>
   <signal>toggled(bool)</signal>
   <receiver>meter</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>metric</sender>
   <signal>toggled(bool)</signal>
   <receiver>bar</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>metric</sender>
   <signal>toggled(bool)</signal>
   <receiver>liter</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>metric</sender>
   <signal>toggled(bool)</signal>
   <receiver>celsius</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>metric</sender>
   <signal>toggled(bool)</signal>
   <receiver>kg</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imperial</sender>
   <signal>toggled(bool)</signal>
   <receiver>feet</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imperial</sender>
   <signal>toggled(bool)</signal>
   <receiver>psi</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imperial</sender>
   <signal>toggled(bool)</signal>
   <receiver>cuft</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imperial</sender>
   <signal>toggled(bool)</signal>
   <receiver>fahrenheit</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imperial</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbs</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>personalize</sender>
   <signal>toggled(bool)</signal>
   <receiver>units_group</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup_5"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_4"/>
 </buttongroups>
</ui>
